application/dash_application/utility/df_manipulation.py

```diff
@@ -290,6 +291,17 @@ def get_bamqc3_and_4():
     )
 
 
+def get_dnaseqqc_and_bamqc4():
```

Comment:
Is calling the function outside of the scope of the ticket?

Reply:
I read it as such. The ticket was more for getting dnaSeqQC into Dashi for future use. Right now, there are no actions for it in [Shesmu](https://shesmu.gsi.oicr.on.ca/olivedash?alert=W10&saved=eyJmaWxlIjoiL2hvbWUvZGViaWFuL3NwYi1zZXF3YXJlLXByb2R1Y3Rpb24vc2hlc211L3Byb2R1Y3Rpb24vbmlhc3NhLWRuYVNlcVFDLnNoZXNtdSIsImxpbmUiOjE2LCJjb2x1bW4iOjYsImhhc2giOiJGRDg3RTE2NjBENTFCNTBCMDU1MEFGNTc4NzdFMDVBM0RCOEMwRUNGIn0&filters=e30)

Reply:
Given that, could this PR please be updated to make the changes in Dashi that are required for the cache to actually be used?